docs/src/main/asciidoc/extension-metadata.adoc

```diff
@@ -50,8 +51,9 @@ metadata:
 <5> Categories the extension should appear under on https://code.quarkus.io[code.quarkus.io]. This field can be omitted, the extension will still be listed on https://code.quarkus.io[code.quarkus.io] but won't be categorized
 <6> Maturity status that could be `stable`, `preview`, `experimental`. It is up to extension maintainers to evaluate the maturity status and communicate it to the users
 <7> Link to the extension guide or documentation page
-<8> https://quarkus.io/guides/extension-codestart[Codestart] information
-<9> Configuration prefix
+<8> Link to an externally hosted image. This is used in the Quarkus dev tools as the extension icon. Alternatively, if you https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/customizing-your-repositorys-social-media-preview[set the social media preview] on your extensions source code repository, the tools will pick up that image.
```

Comment:
should we specify what kind of image? does it assume/recommend/require svg ? or is png etc okey?